### DISCUSSION TITLE
My Brandon421-ops and Brandon423-ops accounts got flagged

### DISCUSSION THREAD

**Discussion by Brandon424-ops** [3kh0/ext-remover#1114] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello everyone I hope you have seen this message my website and my account is gonna be hidden till my account gets unflagged some dumb@ss reported me and idk who did but they ruined everything. Whoever reported me thanks a lot u suck dead@ss.

**Comment by SIMONHSCH** [comment]
😢

**Comment by SIMONHSCH** [comment]
you can choose another name that is not similar to Brandon421-ops

**Comment by S-PScripts** [comment]
> <img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/172834254?s=64&amp;v=4">
> Brandon424-ops

rip

**Reply by S-PScripts** [reply]
probably banned for having more than one account

**Comment by SIMONHSCH** [comment]
You can create your website like [Chrose](https://chrose.netlify.app/?)

**Reply by AshtonDavies** [reply]
> You can create your website like [Chrose](https://chrose.netlify.app/?)

I need to gather up a team or something. A bunch of people could speed it up pretty well rather than him creating his own similar version with copy+pasted details from his old site.

**Reply by SIMONHSCH** [reply]
I can help you for giving you exploits If you want !

**Reply by AshtonDavies** [reply]
Could you join [this Discord server](https://discord.com/invite/FDyT78xUps) and talk about this? I'll try to get a few more people in there maybe.

**Comment by nathblade16** [comment]
this seems to be a problem on github as my freind @guslight4 got flagged, since got unbanned after apealing and now is privatised until he enables 2fa

**Reply by sliyarli** [reply]
how you guys getting your accs unbanned bro it's been days since i wrote to support for my flagged account and all i did was to be victim of a hack attack

**Reply by nathblade16** [reply]
> how you guys getting your accs unbanned bro it's been days since i wrote to support for my flagged account and all i did was to be victim of a hack attack

he had diffrent circumstances where he didn't even do anything, not even get hacked, instead there was a problem with there anti abuse thing

**Comment by FunnyLoaf** [comment, marked as answer]
https://support.github.com/contact/reinstatement



Choose the options that apply, and you might get your account unhidden.
